File: verve patch/base/Applications/CSymphony_0_1/csSamples/CodeExpressionStatement.cs
using System;
using System.CodeDom;
using System.CodeDom.Compiler;
using System.Collections;
using CSymphony.Lib;

namespace CSymphony.CSy
{
	public class CSyUnit : GraphUnit
	{
		public override void Build()
		{
			ns("MyNameSpace");

			declClass(TypePublic, "MyClasss");
			declMethod(MemPublic, "System.Void", "MyMethod");

			MethodRef ConsoleWrite = new MethodRef ("Console", "Write");
			//ex = CodeMethodInvokeExpression

			enter (ex (ConsoleWrite, new Literal ("Example String")));

		}
	}
}

File: verve patch/base/Applications/CSymphony_0_1/vbSamples/CodeExpressionStatement.vb
Option Strict Off
Option Explicit On

Imports System
Imports System.CodeDom
Imports System.CodeDom.Compiler
Imports System.Collections
Imports CSymphony.Lib

Namespace CSymphony.CSy
	Public Class CSyUnit
		 Inherits GraphUnit
		Overrides Public Sub Build()
			ns("MyNameSpace")

			declClass(TypePublic, "MyClasss")
			declMethod(MemPublic, "System.Void", "MyMethod")

			Dim ConsoleWrite As MethodRef  = New MethodRef ("Console", "Write")
			
			enter (ex (ConsoleWrite, newLit("Example String")))
		End Sub
	End Class
End Namespace
			